Rubberduck.CodeAnalysis/Inspections/Concrete/MultipleDeclarationsInspection.cs

```diff
@@ -14,6 +14,23 @@
 
 namespace Rubberduck.Inspections.Concrete
 {
+    /// <summary>
+    /// Flags declaration statements spanning multiple physical lines of code.
```

Comment:
Nope. Copy-pasta I guess. This inspection has nothing to do with spanning multiple lines.